crossing-training.lic

```diff
@@ -103,6 +104,8 @@ class CrossingTraining
 
     check_tithe
 
+    check_attunement
```

Comment:
Could there be a way to opt out of this? I'm not really sure what the advantage of it would be if you already have power walking as part of your training.